packages/design-system/src/components/Dropdown/Dropdown.tsx

```diff
@@ -222,7 +225,7 @@ export const Dropdown: React.FC<DropdownProps> = (props: DropdownProps) => {
       props.inversed && 'ds-c-field--inverse',
       fieldClassName
     ),
-    'aria-label': ariaLabel,
+    'aria-labelledby': `${buttonContentId} ${labelId}`,
```

Comment:
There's something up with this... I'm investigating. Something I noticed is the button seems to read out the word "semicolon" before the button label. I've tried modifying this line to see if it's the culprit (changed it to `FOO ${buttonContentId} ${labelId}`) and my changes aren't announced anywhere in the element. 

I'll push something up once I get to the bottom of it.

Reply:
"semicolon" appears to be read out when the dropdown first loads before a selection is made.